tests/data/validate/validate.test.ts

```diff
@@ -0,0 +1,77 @@
+import Ajv from 'ajv';
+import { cloneDeep } from 'lodash';
+import * as metaschema from '../../../src/commands/data/schemas/util/metaschema.json';
+import { SchemaVerify, TestId } from '../../../src/commands/data/schemas/util/schemaverify';
+import { validSchema } from '../../__helpers__/schemas';
+
+describe('Data - Schema - Validate', () => {
+  const ajv = new Ajv();
+  let schema;
+
+  beforeEach(() => {
+    schema = cloneDeep(validSchema);
+  });
+
+  it('Verifies a Schema', async () => {
+    const validate = new SchemaVerify(ajv, schema, metaschema);
+    const checks = validate.RunChecks();
+    for (const check of checks) {
+      if (check.id === TestId.INPUT_CONDITIONS) {
+        expect(check.ok).toBe(true);
+      }
+    }
+  });
+
+  it('Throws an error when using a property in a transition which is not defined in the schema properties', async () => {
+    delete schema.properties.category;
+
+    const validate = new SchemaVerify(ajv, schema, metaschema);
+    const checks = validate.RunChecks();
+
+    for (const check of checks) {
+      if (check.id === TestId.INPUT_CONDITIONS) {
+        expect(check.ok).toBe(false);
+        expect(check.errors).toStrictEqual([
+          "Transition - mark-as-analyzed : property 'category' is defined in conditions, but not defined in the schema properties",
+        ]);
+      }
+    }
```

Comment:
should we add `expect.assertions(2)` here? 
Just to make sure that they test fail if the checks are removed 😄 
Same for all other tests.

Reply:
I don't mean people removing the expect, that we can't really guard against. I mean if somebody comes along and remove the input-conditions check completely from the code base, these test still pass as they are but they don't do anything anymore.

Reply:
This should make the tests fail but it does not : https://github.com/ExtraHorizon/exh-cli/commit/2a0973caf8fc136b94743ea8aa370a01f0d6b0ce

Reply:
Ah I see what you mean, then I agreeeeeeeee 👍